src/views/landing.njk

```diff
@@ -31,7 +31,8 @@
             text: 'Start now',
             isStartButton: true,
             attributes: {
-              id: 'start-now'
+              id: 'start-now',
```

Comment:
The acceptance criteria mentions adding this to all continue like buttons, do we need to do this?

Reply:
AC is wrong in that regard. It looks CH only sets goal on "start now" button